### PR TITLE
new unearned-income-amount screen

### PR DESCRIPTION
**Pull request by guiburato**
#### 🔗 Pivotal Tracker ticket
[186714238](https://www.pivotaltracker.com/story/show/186714238)

#### ✍️ Description
new unearned-income-amount screen

#### 📷 Design reference
[Figma](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=748-12968&mode=design&t=Mhx1O8zcOqdC2Ct5-0)

#### ✅ Completion tasks

- [ x ] Added relevant tests
- [ x ] Meets acceptance criteria

### CODE REVIEW

**Comment by coltborg** [comment]
Waiting to get [clarification on this design question](https://www.pivotaltracker.com/story/show/186714238/comments/240717848) before approving.

File: src/main/resources/templates/gcc/unearned-income-amount.html [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
       <main id="content" role="main" class="form-card spacing-above-35">
-        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{placeholder}, subtext=#{placeholder})}"/>
+        <th:block th:replace="~{fragments/gcc-icons :: checkIncome}"></th:block>
+        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{unearned-income-amount.header}, subtext=#{unearned-income-amount.subheader})}"/>
         <th:block th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
           <th:block th:ref="formContent">
             <div class="form-card__content">
-                <!-- Put inputs here -->
+              <th:block th:replace="~{fragments/inputs/overrides/money :: money(inputName='unearnedIncomeRental', label=#{unearned-income-amount.rental.income}, postfix=#{unearned-income-amount.posfix})}"/>
+              <th:block th:replace="~{fragments/inputs/overrides/money :: money(inputName='unearnedIncomeUnemployment', label=#{unearned-income-amount.unemployment}, postfix=#{unearned-income-amount.posfix})}"/>
```
Comment (coltborg) [inline src/main/resources/templates/gcc/unearned-income-amount.html:17]: It was not mentioned in the ticket, I've now updated it, that these inputs are dynamic and show based on they checkboxes from the previous screen.

Here's a [link to the Figma comments](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8?type=design&node-id=748-12968&mode=design#747453344).

**Review by coltborg — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/gcc/unearned-income-amount.html (hunk 1 above)
Comment (guiburato) [reply src/main/resources/templates/gcc/unearned-income-amount.html]: @coltborg Thank you for this. I didn't notice it.

**Review by coltborg — APPROVED** [review]
approved!

**Pull request merged**